client-registry-jempi/docker-compose.combined.yml

```diff
@@ -91,8 +91,44 @@ services:
       kafka:
       default:
 
+  jempi-postgresql:
+    image: bitnami/postgresql:15.2.0
+    hostname: postgresql
```

Comment:
We should rather not use hostname here, we can specify this as an env var...
`postgres.server=jempi-postgresql`

See the links below for its usage.

https://github.com/jembi/JeMPI/blob/352501d4eeefa3b894267f41f0581d466b904acf/JeMPI_Apps/JeMPI_API/src/main/java/org/jembi/jempi/AppConfig.java#L41

https://github.com/jembi/JeMPI/blob/352501d4eeefa3b894267f41f0581d466b904acf/JeMPI_Apps/JeMPI_API/src/main/java/org/jembi/jempi/postgres/DbConnect.java#L10

Reply:
cool done